src/client/app/reducers/group.reducer.jsx

```diff
@@ -0,0 +1,32 @@
+let group = {
+  name: 'Loopy Leopards',
+  list: [
+    {
+      name: 'Giancarlo',
+      phone: '123-123-1234'
+    },
+    { 
+      name: 'Joe',
+      phone: '987 merge sort'
+    }
+  ]
+};
+
+export default function group(state = group, action) {
+  switch (action.type) {
+    case 'ADD_TO_GROUP':
+      state.list.push(action.payload);
```

Comment:
Facebook's immutable library will be used for this in one of the upcoming patches.

Thanks for pointing this out.